### ISSUE TITLE
Too much time on creating a web directory

### ISSUE THREAD

**Issue by AmirAJ95** [junyanz/pytorch-CycleGAN-and-pix2pix#1046]
![image](https://user-images.githubusercontent.com/24696018/83197427-506be700-a0f2-11ea-83b0-4a3af66e890b.png)

Hi, 
I tried to run CycleGAN on a very small dataset but as you can see in the above figure it spends a lot of time on creating a web difrectory. (About an hour but still nothing is changed.) Is it normal? (I am using CPU)

**Comment by Dawson557** [comment]
The code actually stops running at this point for me too. I'm using google colab

**Comment by Dawson557** [comment]
welp I found my silly error. I assumed when you continued training it would train for the number of epochs you told it to. Not the number of epochs minus the number of epochs already trained. I was basically telling it to train for zero epochs and confused why it stopped after it created the web directory.

**Comment by rickkk856** [comment]
I had the same issue

Trained for 200 epoch with initial learning rate... than dropout for more 200 epoch
But my computer shutted down on epoch 350 so i needed to run
```
python train.py --dataroot ./datasets/experiment_name --name experiment_name --model pix2pix --netG unet_128  --preprocess none --epoch_count 351 --n_epochs 200 --n_epochs_decay 200 --continue_train
```

Hope this helps other people
